### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by MickeyDLuffy**
please review thoroughly and lemme know

### CODE REVIEW

**Review by jsofosu — DISMISSED** [review]

File: src/app/core/interceptor/cache.interceptor.ts [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    return next.handle(req).pipe(
+      tap(httpEvent =>{
+        if(httpEvent instanceof HttpResponse && httpEvent.ok) {
+            this.cacheService.put(url, httpEvent.body)
+        }
+      })
+    );
+  }
+
+
```
Comment (jsofosu) [inline src/app/core/interceptor/cache.interceptor.ts]: spaces

File: src/app/core/services/book.service.ts [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+})
+export class BookService extends GenericService<BookModel, BookFilter>{
+  protected url = '/books'
+
+
+  constructor( override httpClient: HttpClient) {
+    super(httpClient)
+  }
+
+
```
Comment (jsofosu) [inline src/app/core/services/book.service.ts]: oi spaces

File: src/styles/mixins/_mixins.scss [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+@mixin absolute-center {
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+}
+
+
+
```
Comment (jsofosu) [inline src/styles/mixins/_mixins.scss:23]: why o why

File: src/styles/mixins/_mixins.scss [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+@mixin page-headers {
+    color: colors.$main;
+    padding-bottom: colors.$spacing-1rem;
+    @include type.font(700, 2.5em, 'Lato')
+}
+
+
+
+
```
Comment (jsofosu) [inline src/styles/mixins/_mixins.scss:42]: again

**Review by jsofosu — DISMISSED** [review]

**Review by jsofosu — APPROVED** [review]

**Comment by MickeyDLuffy** [comment]
Judith be careful

**Pull request merged**